README.md

```diff
@@ -37,4 +40,13 @@ branch name: hotfix/FUN-456-ABC-123-bugfix-time
 commit prefix: [ABC-123] [FUN-456]
 ```
 
+# GitHub Action Usage
 
+The primary issue tag extraction step can also be used during GitHub Actions to extract issue tags to use as input for other steps. Default usage as follows:
+
+```yaml
+steps:
+  - name: Extract Issue Tags
+    id: issues-default # usage: ${{ steps.issues-default.outputs.issue-tags }}
+    uses: Renddslow/prepare-branch-commit@v1
```

Comment:
@Renddslow Before you merge this, can you update the tag here to match what you'll tag this repo with?